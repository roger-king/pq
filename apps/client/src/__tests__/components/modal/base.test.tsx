import React from 'react';
import renderer from 'react-test-renderer';

import Modal from '../../../components/modal';

describe('<Modal />', () => {
  test('should display Base Modal', async () => {
    const tree = renderer
      .create(
        <Modal title="Testing Modal" onClose={() => console.log('testing')}>
          {' '}
          <div>Some content for the base model</div>
        </Modal>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
