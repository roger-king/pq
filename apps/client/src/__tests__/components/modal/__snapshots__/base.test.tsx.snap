// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Modal /> should display Base Modal 1`] = `null`;
